data/json/npcs/talk_tags.json

```
@@ -474,7 +546,10 @@
       "Hasta luego, <name_g>!",
       "I'm outta here! <done_mugging>",
       "Bye bye, <name_b>!",
-      "Thanks, <name_g>!"
+      "Thanks, <name_g>!",
+      "Pleasure doing business with you.",
+      "Now beat it, you <name_b>",
+      "Good haul. See you around."
```

```suggestion
      "Good haul. See you around<punc>"
```
Good opportunity to use <punc> here.

---

Didn't find "punc" anywhere. I never worked on anything here and it doesn't exactly come with instructions, all I have to go on is what's written in the category.